tutorecommerce/patches/openedx-lms-production-settings

```diff
@@ -1,6 +1,8 @@
 # Ecommerce
 ECOMMERCE_PUBLIC_URL_ROOT = "{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ ECOMMERCE_HOST }}"
 ECOMMERCE_API_URL = ECOMMERCE_PUBLIC_URL_ROOT + "/api/v2"
+{% if ECOMMERCE_MFE_APP %}
 ORDER_HISTORY_MICROFRONTEND_URL = "{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ MFE_HOST }}/{{ ECOMMERCE_MFE_APP['name'] }}/orders"
+{% endif %}
 MFE_CONFIG["ECOMMERCE_BASE_URL"] = ECOMMERCE_PUBLIC_URL_ROOT
 MFE_CONFIG["ORDER_HISTORY_URL"] = ORDER_HISTORY_MICROFRONTEND_URL
```

Comment:
@Squirrel18  Sorry I didnt see this undefined issue before..

![image](https://github.com/Pearson-Advance/tutor-ecommerce/assets/40271196/c0b60cec-fa26-469c-a4d1-394258250b73)

```suggestion
MFE_CONFIG["ORDER_HISTORY_URL"] = ""
{% if ECOMMERCE_MFE_APP %}
ORDER_HISTORY_MICROFRONTEND_URL = "{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ MFE_HOST }}/{{ ECOMMERCE_MFE_APP['name'] }}/orders"
MFE_CONFIG["ORDER_HISTORY_URL"] = ORDER_HISTORY_MICROFRONTEND_URL
{% endif %}
MFE_CONFIG["ECOMMERCE_BASE_URL"] = ECOMMERCE_PUBLIC_URL_ROOT
```